extensions/ql-vscode/src/variant-analysis/variant-analysis-manager.ts

```diff
@@ -374,7 +380,12 @@ export class VariantAnalysisManager
       this.dbManager,
     );
 
-    const queryName = getQueryName(queryMetadata, queryFile);
+    const firstQueryFile = qlPackDetails.queryFiles[0];
```

Comment:
Getting the metadata and query text of the first query is a bit of a hack to fit in with the existing data format. Do we want to mark it as such with a comment? We don't plan to be addressing this any time soon so it could sit like this for a while and we'll forget it's like this.

Reply:
Ah yes, I was planning to leave a comment but forgot!